tests/Unit/Jobs/SendRemindersTest.php

```diff
@@ -0,0 +1,118 @@
+    //         'default_time_reminder_is_sent' => '07:00',
+    //         'has_access_to_paid_version_for_free' => 1,
+    //     ]);
+    //     $contact = factory('App\Contact')->create(['account_id' => $account->id]);
+    //     $user = factory('App\User')->create(['account_id' => $account->id]);
+    //     $reminder = factory('App\Reminder')->create([
+    //         'account_id' => $account->id,
+    //         'contact_id' => $contact->id,
+    //         'next_expected_date' => '2017-01-01',
+    //     ]);
+
+    //     $exitCode = Artisan::call('send:reminders', []);
+
+    //     Bus::assertDispatched(SendReminderEmail::class);
+    //     Bus::assertDispatched(SetNextReminderDate::class);
+    // }
+
+    // public function test_it_deletes_the_reminder_if_contact_doesnt_exist()
+    // {
+    //     Bus::fake();
+
+    //     Carbon::setTestNow(Carbon::create(2017, 1, 1, 7, 0, 0));
+
+    //     $account = factory('App\Account')->create([
+    //         'default_time_reminder_is_sent' => '07:00',
+    //         'has_access_to_paid_version_for_free' => 1,
+    //     ]);
+    //     $user = factory('App\User')->create(['account_id' => $account->id]);
+    //     $reminder = factory('App\Reminder')->create([
+    //         'account_id' => $account->id,
+    //         'contact_id' => 3,
+    //         'next_expected_date' => '2017-01-01',
+    //     ]);
+
+    //     $exitCode = Artisan::call('send:reminders', []);
+
+    //     Bus::assertNotDispatched(SendReminderEmail::class);
+    //     Bus::assertNotDispatched(SetNextReminderDate::class);
+
+    //     $this->assertDatabaseMissing('reminders', [
+    //         'id' => $reminder->id,
+    //     ]);
+    // }
+
+    // public function test_it_schedules_multiple_emails_jobs_but_only_one_set_next_reminder_job()
+    // {
+    //     Bus::fake();
+
+    //     Carbon::setTestNow(Carbon::create(2017, 1, 1, 7, 0, 0));
+
+    //     $account = factory('App\Account')->create([
+    //         'default_time_reminder_is_sent' => '07:00',
+    //         'has_access_to_paid_version_for_free' => 1,
+    //     ]);
+    //     $contact = factory('App\Contact')->create(['account_id' => $account->id]);
+    //     $user1 = factory('App\User')->create(['account_id' => $account->id]);
+    //     $user2 = factory('App\User')->create(['account_id' => $account->id]);
+    //     $reminder = factory('App\Reminder')->create([
+    //         'account_id' => $account->id,
+    //         'contact_id' => $contact->id,
+    //         'next_expected_date' => '2017-01-01',
+    //     ]);
+
+    //     $exitCode = Artisan::call('send:reminders', []);
+
+    //     Bus::assertDispatched(SendReminderEmail::class, 2);
+    //     Bus::assertDispatched(SetNextReminderDate::class, 1);
+    // }
+
+    public function test_it_doesnt_schedule_email_if_on_unpaid_plan()
+    {
+        Bus::fake();
+
+        config(['monica.requires_subscription' => true]);
```

Comment:
@asbiin this is what's not working.

`'monica.requires_subscription'` is set to true, but once we are in the `send:reminders` function, it's set to false. It's like this config is not persistent.

Reply:
It's working now